Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -208,8 +208,8 @@ open class CarPlayMapViewController: UIViewController {
         
         navigationMapView.userLocationStyle = .puck2D()
         
-        navigationMapView.mapView.ornaments.options.logo._visibility = .hidden
-        navigationMapView.mapView.ornaments.options.attributionButton._visibility = .hidden
+        navigationMapView.mapView.ornaments.options.logo.visibility = .hidden
+        navigationMapView.mapView.ornaments.options.attributionButton.visibility = .hidden
```

Comment:
We’re going through some lengths to keep hiding attribution on CarPlay, even though the `visibility` property is now `Restricted` SPI, not intended for customer use. Unfortunately, we don’t really have a choice, because the map SDK only provides an attribution control that requires interactivity. mapbox/mapbox-maps-ios#745 proposes an option to add an unobtrusive textual attribution label to the corner of the map.

In the meantime, the user can still access attribution through the phone screen, assuming the application configures CarPlay navigation in the usual way.